src/txn/txn.c

```diff
@@ -467,16 +467,6 @@ __txn_oldest_scan(WT_SESSION_IMPL *session, uint64_t *oldest_idp, uint64_t *last
     }
     WT_STAT_CONN_INCRV(session, txn_sessions_walked, i);
 
-    /*
-     * Kludge: layered tables need to keep transaction IDs alive to manage garbage collection in the
-     * ingest table. If the layered table server is running, retrieve the oldest ID necessary in all
-     * layered ingest tables and use that as the oldest reader.
-     */
-    if (FLD_ISSET(conn->server_flags, WT_CONN_SERVER_LAYERED)) {
```

Comment:
Per my comment above about transaction IDs being relevant for the ingest table - should this stay?

Reply:
Can you give a scenario that you think we need to consider transaction IDs?

Reply:
I don't understand how transaction ids are useful to decide which content we should include across nodes.